shared/whats-new/versions.tsx

```diff
@@ -83,15 +83,15 @@ export const Current = ({seen, onNavigate, onNavigateExternal}: VersionProps) =>
 export const Last = (_: VersionProps) => {
   return (
     <Version>
-      <VersionTitle title="Last Release" />
+      <VersionTitle title="Last release" />
     </Version>
   )
 }
 
 export const LastLast = (_: VersionProps) => {
   return (
     <Version>
-      <VersionTitle title="Last Last Release" />
+      <VersionTitle title="Last last release" />
```

Comment:
Is this seen by users?

Reply:
It will when there are three versions. Since there is only one, only `Current` is rendered.